src/main/webapp/js/calendar.js

```diff
@@ -13,8 +13,90 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// TODO(savsa): Add ability to remove courses from the calendar
+// TODO(#34): Add ability to choose which section the user wants, not just
+//  automatically choosing the first section
+
+let id = 1;  // The id of the next schedule that will be added to the calendar
+let calendar;     // The calendar object
+const DateTime = luxon.DateTime;  // An alias for the Luxon DateTime object
+/**
+ * Days of the school week.
+ * @enum {int}
+ */
+const enumDays = {
+  DATE_SUNDAY: 0,
+  DATE_MONDAY: 1,
+  DATE_TUESDAY: 2,
+  DATE_WEDNESDAY: 3,
+  DATE_THURSDAY: 4,
+  DATE_FRIDAY: 5,
+  DATE_SATURDAY: 6,
+};
+
+export const Calendar = (() => {
+  /**
+   * Adds a course to the calendar.
+   * @param {Object} course The JSON Object for the course.
+   */
+  async function addCourse(course) {
```

Comment:
please move addCourse out from `export` too. Is there a reason it needs to be in here?